docs/NewFlux_Comentary.md

```diff
@@ -0,0 +1,252 @@
+# New Flux
+
+Author: Nathaniel Cook
+
+Recently Paul Dix and I added a document on the vision of a new Flux that is more usable.
+The design there solves several key points of feedback we have received on Flux.
+
+
+That document describe the what and some of the why. This document will discuss the how.
+This document is not meant to be prescriptive rather just my personal commentary on how I see the vision of new Flux could be implemented.
+It is meant to be a seed to start a good team discussion around how  we can implement the new Flux vision.
+
+
+## What is new?
+
+First before we talk about how we would implement the new vision, we should clarify what is new.
+
+### Single Table vs Stream of Tables
+
+The new Flux data model changes to be a single table with a homogeneous schema.
+This is different from the current design which represents data as a stream (unbounded sequence) of tables with possibility differing schemas.
+
```

Comment:
This eliminates group keys in favor of a schema? How does this change the way that users are able to group and chart data for comparison?

Reply:
User's can still group their data, but instead of that grouping being persisted across different functions/transformation the grouping only _lasts_ as long at the function that used it. For example the `aggreate` function will group the data, compute the aggregate of each result and then output a single table with the results from each group. After the call to `aggregate` the grouping is no longer relevant/important.